spec/forms/candidate_interface/gcse_qualification_details_form_spec.rb

```diff
@@ -149,5 +171,56 @@
         expect(qualification.award_year).to eq('1990')
       end
     end
+
+    describe '.build_from_qualification' do
+      context 'with the international_gcses feature flag off' do
+        let(:data) do
+          {
+            grade: 'D',
+            award_year: '2012',
+          }
+        end
+
+        it 'creates an object based on the provided ApplicationForm' do
+          qualification = ApplicationQualification.new(data)
+          gcse_details_form = CandidateInterface::GcseQualificationDetailsForm.build_from_qualification(qualification)
+
+          expect(gcse_details_form).to have_attributes(data)
+        end
+      end
+
+      context 'with the international_gcses feature flag on, the qualification_type in non_uk and grade is not_applicable' do
```

Comment:
are you able to get away with another level of nesting for the feature flag?

Reply:
Afraid not. It's already at 4/4.